src/Apia/Options.hs

```diff
@@ -46,10 +45,12 @@ module Apia.Options
            , optWithIleanCoP
            , optWithMetis
            , optWithSPASS
+           , optWithtptp2X
```

Comment:
This is  weird, I can actually see the commit that you did about that, I pulled not so long ago, and this were the last commit: 
```bash
commit fe58c1c9aeaf9c02d598918421197698fbdc9ffd
Author: Andrés Sicard-Ramírez <asr@eafit.edu.co>
Date:   Wed Jul 20 16:11:44 2016 -0500

    Supported TPTP 6.4.0.

commit cbf5cad8bb7ee92c8eaf46de7dff583b512cd437
Author: Andrés Sicard-Ramírez <asr@eafit.edu.co>
Date:   Wed Jul 20 15:12:05 2016 -0500

    Using `tptp4X` instead of `tptp2X`.
```bash

What should I do?